cloud-minecraft-extras/build.gradle.kts

```diff
@@ -17,6 +19,8 @@ dependencies {
     }
     compileOnlyApiAndTests(libs.adventureApi)
     compileOnlyApiAndTests(libs.adventureTextSerializerPlain)
+    compileOnlyApiAndTests(libs.adventureTextSerializerLegacy)
```

Comment:
These should just be compileOnly, not compileOnlyApi. Should also add a comment that these are for ComponentArgument annotation mapping